cristin-person/src/test/java/no/unit/nva/cristin/person/employment/update/UpdatePersonEmploymentHandlerTest.java

```diff
@@ -159,7 +159,7 @@ private InputStream createRequest(String body) throws JsonProcessingException {
                    .withBody(body)
                    .withCurrentCustomer(customerId)
                    .withTopLevelCristinOrgId(TOP_ORG_ID)
-                   .withAccessRights(customerId, EDIT_OWN_INSTITUTION_USERS)
+                   .withAccessRights(customerId, MANAGE_OWN_AFFILIATION)
```

Comment:
MANAGE_OWN_AFFILIATION? in the docs it says MANAGE_OWN_AFFILIATIONS, which is slightly less confusing